net-misc/curl/curl-7.48.0.recipe

```diff
@@ -1,16 +1,18 @@
 SUMMARY="A commandline-tool and library for downloading data from URLs"
-DESCRIPTION="Curl is a command line tool for transferring data with URL \
+DESCRIPTION="curl is a command line tool for transferring data with URL \
 syntax, supporting DICT, FILE, FTP, FTPS, Gopher, HTTP, HTTPS, IMAP, IMAPS, \
-LDAP, LDAPS, POP3, POP3S, RTMP, RTSP, SCP, SFTP, SMTP, SMTPS, Telnet and TFTP. curl supports SSL certificates, HTTP POST, HTTP PUT, FTP uploading, HTTP form \
+LDAP, LDAPS, POP3, POP3S, RTMP, RTSP, SCP, SFTP, SMTP, SMTPS, Telnet and TFTP.
+curl supports SSL certificates, HTTP POST, HTTP PUT, FTP uploading, HTTP form \
 based upload, proxies, cookies, user+password authentication (Basic, Digest, \
 NTLM, Negotiate, kerberos...), file transfer resume, proxy tunneling and a \
 busload of other useful tricks."
-HOMEPAGE="http://curl.haxx.se"
+HOMEPAGE="https://curl.haxx.se/"
 COPYRIGHT="1996-2016, Daniel Stenberg"
 LICENSE="Curl"
 REVISION="1"
-SOURCE_URI="http://curl.haxx.se/download/curl-$portVersion.tar.bz2"
-CHECKSUM_SHA256="ddc643ab9382e24bbe4747d43df189a0a6ce38fcb33df041b9cb0b3cd47ae98f"
+SOURCE_URI="https://curl.haxx.se/download/curl-$portVersion.tar.bz2"
+#CHECKSUM_SHA256="ddc643ab9382e24bbe4747d43df189a0a6ce38fcb33df041b9cb0b3cd47ae98f"
```

Comment:
No, it's just that I first want to rebuild 7.47.1 locally with the new package split (which includes 2 arch-independent "common" and "common_devel" packages) to make sure future upgrades won't be affected by the new split layout.
So I'll install locally all 7.47.1 packages, then play with pkgman install and uninstall, and then try to upgrade all or a subset. Once I see that upgrades are OK with the new split layout I'll drop the SHA256 for 7.47.1 and put a nice commit message for directly 7.48.0. I'll also check that the upgrade from 7.45.0 to 7.48.0 works fine.
(The reason why I pushed it in WIP state was to save others a duplicate job ;-)